src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -795,6 +796,8 @@ trait BeamHelper extends LazyLogging {
 
     new UrbanSimScenarioSource(
       scenarioFolder = beamConfig.beam.exchange.scenario.folder,
+      isWarmMode = beamConfig.beam.warmStart.enabled,
+      planFile = matsimConfig.plans.getInputFile,
```

Comment:
in this case is loading the wrong file. `matsimConfig.plans` points to xml and urbansim is supposed to load to csv/parquet files